source/cli.rst

```diff
@@ -0,0 +1,432 @@
+GLPI command-line interface
+===========================
+
+GLPI includes a CLI tool to help you to manage your GLPI instance.
+This interface is provided by the `bin/console` script which can be run from the root of your GLPI directory.
+
+glpi:assets:cleansoftware
+-------------------------
+
+Aliases: `assets:cleansoftware`
+
+Description
+***********
+
+Remove software versions with no installation and software with no version
+
+
+glpi:build:compile_scss
+-----------------------
+
+Aliases: `build:compile_scss`
+
+Description
+***********
+
+Compile SCSS file.
+
+
+glpi:cache:clear
+----------------
+
+Aliases: `cache:clear, glpi:system:clear_cache, system:clear_cache`
+
+Description
+***********
+
+Clear GLPI cache.
+
+
+glpi:cache:clear
```

Comment:
Duplicate, already documented line 29.

Reply:
Sounds like this is not specific to current PR which includes several other that were waiting. @cconard96 could you please check that, and I guess you can rebase current PR

Reply:
No, it is related to #67. Odd that it shows up twice when the command documentation was generated automatically.
I'll regenerate the CLI command document page later and see if it is still duplicated.